### ISSUE TITLE
⛔ 🔗 Exit to wordpress doesn't work when editor top bar feature is enabled (Duplicate of #22828)

### ISSUE THREAD

**Issue by barla-x88** [elementor/elementor#25465]
### Prerequisites

- [x] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

I have configured elementor to take me to the wp dashboard on exit. This works as expected when the editor top bar feature is inactive but when this feature is active, exit to wordpress option takes me to the post edit page instead of dashboard. 

### Steps to reproduce

1. Install Elementor 
2. Enable editor top bar feature
4. configure ` exit to` option in user preference to 'wp dashboard'
5. click on elementor logo 
6. click on exit to wordpress

### Expected behavior

1. I'm expecting to go to wordpress dashboard when I click exit to wordpress

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: MySQL Community Server - GPL v8.2.0
	PHP Version: 8.2.14
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.3
	Site URL: http://localhost
	Home URL: http://localhost
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.0.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.20.1
		Author: Elementor.com


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Upgrade Swiper Library: Active by default
	Grid Container: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active by default
	Build with AI: Active by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Optimize Image Loading: Active by default
	Pages Panel: Inactive by default


== Log ==
	


== Elementor - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
I can confirm this bug

**Comment by nicholaszein** [comment]
Hi, @barla-x88,

Thank you for reaching out! 🙏 

### ⛔ After reviewing your report we concluded this is not a bug caused by Elementor. The feature you're referring to is not yet supported.

If you think this should be an Elementor feature, please cast your ⬆️ **Upvote** in #22828 and we will consider it in the future if there is enough demand (votes) from other users.

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
